src/api/roles/coach/coach.py

```
@@ -65,9 +66,83 @@ def create_coach_request(coach_details: CoachRequestInput, db = Depends(get_sess
+        for c in new_coach_details.certifications:
+            db.add(CoachCertifications(coach_id=coach.id, certification_id=c.id)) # type: ignore
+    
+    if new_coach_details.experiences is not None:
+        db.query(CoachExperience).filter(Experience.id == coach_experiences_id).delete(synchronize_session=False)
+        for e in new_coach_details.experiences:
+            db.add(CoachExperience(coach_id=coach.id, experience_id=e.id)) # type: ignore
+
+    db.flush()
+    db.commit()
+
+    return UpdateCoachInfoResponse(coach_request_id=cr.id, coach_id=coach_acc.id) # type: ignore
```

Ruff (0.15.9)

[error] Undefined name `cr`

(F821)